.editorconfig

```diff
@@ -6,3 +6,6 @@ insert_final_newline = true
 charset = utf-8
 indent_style = space
 indent_size = 2
+
+[{*.hbs.html,*.js}]
+indent_size = 0 # disable indentation for handlebars and js files, they use eslint and djlint
```

Comment:
what's the problem in leaving `indent_size = 2`? isn't it better than nothing for people with editors without eslint configured?

Reply:
The problem with the editorconfig indentation is that it clashes with the current eslint/djlint configuration
an example would be the alignment of the properties in an hbs.html:
```
<div class=""
        name=""
```
You're not guaranteed that `name` would be aligned on a multiple of 2.

Since eslint and djlint enforce it on their own configuration, disabling it in the editorconfig does not change a thing, unless you do not activate them.

Reply:
Non-fixed indents is a bad practice, so we should rather make sure we don't have these. :)

Reply:
I don't think it is a non-fixed indent but rather an alignment on the properties, a more complex example would be
```<div class=""
     name=""
     id=""
     style=""
<div/>
<super-long-name class=""
                 style=""
...
```

Reply:
whatever the terminology, the good practice is what prettier does by splitting the line into one attribute per line, all indented based on the same indentation width:

```
<div
  class=""
  name=""
  id=""
></div>
```